libs/s25main/mapGenerator/RandomMap.cpp

```diff
@@ -220,6 +211,8 @@ namespace rttr { namespace mapGenerator {
         const auto mountainLevel = LimitFor(map_.z, land, static_cast<uint8_t>(map_.height.minimum + 1)) + 1;
         const auto rivers = CreateRivers(center);
 
+        CreateFreeIslands(std::count(map_.z.begin(), map_.z.end(), map_.height.minimum));
```

Comment:
This is where appveyor is warning. I admit that this here is a pointless warning and I've never understood why std::count returns a signed value. But introducing a variable and a cast here does make sense. It isn't immediately obvious what this counts, so a named variable helps